.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: build
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+env:
+  PACKAGE_DIR: adbpyg_adapter
+  TESTS_DIR: tests
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ["3.7", "3.8", "3.9"]
```

Comment:
```suggestion
        python: ["3.7", "3.8", "3.9", "3.10"]
```

Reply:
addressed in https://github.com/arangoml/pyg-adapter/pull/2/commits/2158581d3daa4e59ff1e5c87499deb5040d64200